validation.Rmd

```diff
@@ -129,22 +167,31 @@ Before seeing how these metrics can easily be implemented in code, we will defin
 
 ### Root mean squared error
 
-RMSE is similar to the absolute erro metrics, but uses the *sum of squares* of the error. Recent work in geoscience has suggested that RMSE is preferable to absolute measures of error when the errors approximate a normal distribution (Chai and Draxler 2014).
+RMSE is similar to the absolute error metrics, but uses the *sum of squares* of the error. Recent work in geoscience has suggested that RMSE is preferable to absolute measures of error when the errors approximate a normal distribution (Chai and Draxler 2014).
 Errors in spatial microsimulation tend to have a normal-ish distribution, with many very small errors around the mean of zero and comparatively few larger error. RMSE is defined as follows:
 
 $$
-RMSE = \sqrt{frac{i}{n} \sum_i e^2_i}
+RMSE = \sqrt{\frac{1}{n} \sum_i^n e^2_i}
 $$
 
-```{r, echo=FALSE}
-# As with TAE, there is also a standardised version of RMSE (), normalised root mean error squared (NRMSE). This is calculated by dividing RMSE by the range of the observed values:
-# $$
-# NRMSE = frac{RMSE}{max(obs) - min(obs)} 
-# $$
-```
+RMSE is an interesting measure of the error, since SAE would be the same if the errors are $(1,1,1,1)$ or
+$(0,0,0,4)$. However, we consider the fit as globally better if it contains several few errors than if it is perfect for 3 zones and a higher error for the fourth. In this case, RMSE will detect this difference. 
+For the first case, RMSE is $ \sqrt{\frac{4}{4}} $. For the second case, RMSE equals $\sqrt{\frac{4^2}{4}}=2 $.
```

Comment:
NB: in inline maths, never leave a space after the $.

"$ \sqrt" made the build fail, it took me ages to figure out why!

Changed to "$\sqrt" in latest commit.